src/leaspy/algo/base.py

```diff
@@ -47,31 +51,31 @@ class AlgorithmName(str, Enum):
 class AbstractAlgo(ABC):
     """
     Abstract class containing common methods for all algorithm classes.
-    These classes are child classes of `AbstractAlgo`.
+    These classes are children of `AbstractAlgo`.
 
     Parameters
     ----------
-    settings : :class:`.AlgorithmSettings`
-        The specifications of the algorithm as a :class:`.AlgorithmSettings` instance.
+    settings : :class:`~leaspy.algo.AlgorithmSettings`
+        The specifications of the algorithm as a :class:`~leaspy.algo.AlgorithmSettings` instance.
 
     Attributes
     ----------
-    name : str
+    name : :obj:`str`
         Name of the algorithm.
-    family : str
+    family : :obj:`str`
         Family of the algorithm. For now, valid families are:
```

Comment:
We don't need to describe all possible values because the enumeration is doing it for us already:

```suggestion
        Family of the algorithm.
```